### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by Mosgoss** [yoyzo/arab#11]
In cloudstream app Shahid4u url needs an update .
Thanks

**Comment by ImZaw** [comment]
Not sure if it does because i basically dont use the app. Ill have to check when i have free time

**Comment by Waleed-titi** [comment]
I don't think that shahid4u2 is working and it won't work cause the site is closed by the owner cause the authorities in Egypt arrest the site's owner .
